### ISSUE TITLE
Popover box shadow does not meet 3:1 color contrast

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#21478]
Is this an actual problem that needs to be solved ?

**Comment by tudorpopams** [comment]
[BacklogReview] This needs to be discussed with the design team and make sure it gets adjusted as it might affect everyone using Popover, especially in a11y-related automated tests.
